website/allowed-subcategories.txt

```diff
@@ -81,6 +81,7 @@ Lex
 License Manager
 Lightsail
 MQ
+Macie2
```

Comment:
As noted in https://github.com/hashicorp/terraform-provider-aws/pull/19069/files#r619495523, the category should be `Macie` for Macie v2 (and `Macie Classic` for v1). This should be removed.